hedera-node/hedera-app/src/main/java/com/hedera/node/app/grpc/impl/MethodBase.java

```diff
@@ -143,7 +144,9 @@ public void invoke(
             callsHandledSpeedometer.cycle();
         } catch (final Exception e) {
             // Track the number of times we failed to handle a call
-            logger.error("Possibly CATASTROPHIC failure while handling a GRPC message", e);
+            if (!(e instanceof StatusRuntimeException)) {
+                logger.error("Possibly CATASTROPHIC failure while handling a GRPC message", e);
+            }
```

Comment:
Thinking about this some more, I wonder if we should change the error message to something like "Unexpected Exception while handling GRPC message." What do you think? (Should not block this PR.)

Reply:
Yes, that would be less scary :-)